docs_src/microservices/configuration/V3MigrationCommonConfig.md

```diff
@@ -1,11 +1,115 @@
-Changed database configuration from `Databases map[string]bootstrapConfig.Database` to `Database bootstrapConfig.Database`
+As stated in the top level V3 Migration guide, common configuration has been separated out from each service's private configuration. See the [Service Configuration](../CommonConfiguration/) page for more details on the new **Common Configuration**.
 
-!!! exmaple "Example V3 Database configuration"
+There have also been changes to some sections of the common configuration in order to make them consistent and stream-lined for all EdgeX services 
+
+## MessageBus
+
+In EdgeX 3.0 the EdgeX MessageBus configuration has been refactored and renamed to be `MessageBus`. Prior to EdgeX 3.0, Core/Support Services and Device services had it as `MessageQueue` and Applications Services had it as `MessageBus` under the `Trigger` configuration. Now all services have it as top level `MessageBus`. In addition to the rename, the following fields have been add or removed:
+
+### New Settings
+
+- **Disabled** - Defaults to `false`. Set to `true` by Application Services that don't need the EdgeX MessageBus for Trigger or Metrics. When set to `false` this allows for Metrics to still be published to the EdgeX MessageBus when the Trigger is set to `http` or `external-mqtt`
+- **BaseTopicPrefix** - Base topic prefix that is prepended to all the new topic constants. Defaults to `edgex` if not set.
+
+### Removed Settings
+
+- **PublishTopicPrefix** - Topics are no longer configurable, except for `BaseTopicPrefix`
+- **SubscribeTopic** - Topics are no longer configurable, except for `BaseTopicPrefix`
+- **Topics** - Topics are no longer configurable, except for `BaseTopicPrefix`
+- **SubscribeEnabled** - No longer needed by Core Data. If Core Data's `PersistData` is set to`true` the Core Data will always subscribe to events from the EdgeX MessageBus
+
+### Custom MessageBus configuration
+
+If your deployment has customized any of the EdgeX provided service's `MessageBus` configuration, you will need to re-apply your customizations to the EdgeX 3.0 version of the service's `MessageBus` configuration in the new separated out common configuration.
+
+!!! example - "Example V3 MessageBus configuration - Common"
+    ```yaml
+      MessageBus:
+        Protocol: "redis"
+        Host: "localhost"
+        Port: 6379
+        Type: "redis"
+        AuthMode: "usernamepassword"  # required for redis MessageBus (secure or insecure).
+        SecretName: "redisdb"
+        BaseTopicPrefix: "edgex" # prepended to all topics as "edgex/<additional topic levels>
+        Optional:
+          # Default MQTT Specific options that need to be here to enable environment variable overrides of them
+          Qos:  "0" # Quality of Service values are 0 (At most once), 1 (At least once) or 2 (Exactly once)
+          KeepAlive: "10" # Seconds (must be 2 or greater)
+          Retained: "false"
+          AutoReconnect: "true"
+          ConnectTimeout: "5" # Seconds
```

Comment:
i thought we made this into time duration like "5s"? no?

Reply:
No, that is an MQTT setting, which requires int.